CRM/Admin/Form/Setting/Localization.php

```diff
@@ -184,7 +184,7 @@ public function postProcess() {
 
     //cache contact fields retaining localized titles
     //though we changed localization, so reseting cache.
-    CRM_Core_BAO_Cache::deleteGroup('contact fields');
+    Civi::cache('fields')->flush();
```

Comment:
note that I'm not calling resetCaches() here unlike in the other places where i am replacing deleteGroup with flush because the navigation deleteGroup will still call resetCaches (and is included when navigation is converted)